ctapipe/image/tests/test_extractor.py

```diff
@@ -617,3 +635,46 @@ def test_global_peak_window_sum_with_pixel_fraction(subarray):
 
     expected = np.average([29, 30, 31], weights=[5, 10, 3])
     assert np.allclose(dl1.peak_time[bright_pixels], expected / sample_rate)
+
+
+def test_flashcam_extractor(toymodel_1_MST_FC, prod5_gamma_simtel_path):
```

Comment:
I'd prefer a test similar to the other ones on artificial input, not on an actually simulated event, where you test the different options result in the expected outcome, which is hard to do on simulated events without a "truth" for the waveform / charge including noise / peak time.

Reply:
At least additionally. Testing that it works by comparing with a simulated true image is also fine, but you needed to specify a rather high threshold and tolerance, which makes this test brittle / not very sensitive to actual bugs.